pkg/controller/account/account_controller.go

```diff
@@ -716,15 +757,28 @@ func (r *ReconcileAccount) assumeRole(
 }
 
 func (r *ReconcileAccount) initializeRegions(reqLogger logr.Logger, currentAcctInstance *awsv1alpha1.Account, creds *sts.AssumeRoleOutput, regionAMIs map[string]awsv1alpha1.AmiSpec) error {
+	// Determine if in fedramp env
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		log.Error(err, "Failed retrieving configmap")
+	}
+	ifFedramp, err := controllerutils.IsFedramp(configMap)
+	if err != nil {
+		log.Error(err, "Unable to verify if cluster is fedramp")
+	}
```

Comment:
do we really need to read the configmap again?
If so, wouldn't we want to return an error if we can't read it?
I see these lines duplicated multiple times in the PR, probably worth encapsulating in their own function?